src/main/java/com/github/redouane59/twitter/TwitterClient.java

```diff
@@ -694,8 +722,26 @@ public CollectionsResponse collectionsEntries(final String collectionId, int cou
 
   public static TwitterCredentials getAuthentication() {
     String credentialPath = System.getProperty("twitter.credentials.file.path");
+    if(credentialPath!=null) return getAuthentication(new File(credentialPath));
```

Comment:
It's default code style for Eclipse. Can check discrepancies.

Reply:
Yes if you could use the project code style it would be perfect to have the same style everywhere :)
(I'm using intellij but with a custom Google Style xml)

Reply:
@redouane59 , I tried to load those custom code style (https://github.com/redouane59/twittered/blob/develop/.idea/codeStyles/GoogleStyle.xml) to Eclipse - and, if shortly, no luck. If you don't mind: can you please reformat that by yourself in IntelliJ IDEA?

Reply:
Yes no problem I'll do it :)

Reply:
By the way it looks tricky to commit something on a branch outside my repository... Should I fork your fork haha ? Otherwise i'll do it on another commit after this was merged.

Reply:
I didn't try to change PRs from others - yea - I think that modification after merge will work:)